Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -1263,4 +1275,237 @@ lemma nonsingularLift_addMap {P Q : PointClass F} (hP : W.NonsingularLift P)
 
 end Addition
 
+section Point
+
+/-! ### Nonsingular rational points -/
+
+variable (W') in
+/-- A nonsingular rational point on `W'`. -/
+@[ext]
+structure Point where
+  {point : PointClass R}
```

Comment:
Can't you add the docstring here directly?

Reply:
You mean the docstring in `add_decl_doc` right after this? It doesn't seem to compile for me.

Reply:
Maybe indentation problem? I made it work, see commit to be pushed.